src/index.js

```diff
@@ -6,8 +6,7 @@ const mafmt = require('mafmt')
 const { EventEmitter } = require('events')
 const debug = require('debug')
 
-const log = debug('libp2p:bootstrap')
-log.error = debug('libp2p:bootstrap:error')
+const error = debug('libp2p:bootstrap:error')
```

Comment:
For keeping consistent with other libp2p repos, I suggest changing this to:

```js
const debug = require('debug')
const log = Object.assign(debug('libp2p:bootstrap'), {
  error: debug('libp2p:bootstrap:error')
})
```

What do you think? We currently do not use the the normal log, but perhaps we should log when the bootstrap starts.

Reply:
This makes sense.  I added a log message to the `start` method in my latest commit.